dlp/api/Snippets/DeidentifyWithFpe.cs

```diff
@@ -0,0 +1,84 @@
+﻿// Copyright (c) 2020 Google LLC.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+
+// [START dlp_deidentify_fpe]
+
+using Google.Api.Gax.ResourceNames;
+using Google.Cloud.Dlp.V2;
+using Google.Protobuf;
+using System;
+using System.Collections.Generic;
+using static Google.Cloud.Dlp.V2.CryptoReplaceFfxFpeConfig.Types;
+
+public class DeidentifyWithFpe
+{
+    public static DeidentifyContentResponse Deidentify(
+        string projectId,
+        string dataValue,
+        IEnumerable<InfoType> infoTypes,
+        string keyName,
+        string wrappedKey,
+        string alphabet)
```

Comment:
This shouldn't be a string, it should be FfxCommonNativeAlphabet, and then there's no need to parse the value. This is leftover from the command line runner. Change here and everywhere that proper types are used instead of string